lib/grammars.coffee

```diff
@@ -141,12 +141,20 @@ module.exports =
       args: (context) -> [context.filepath]
 
   Java:
-    "File Based":
-      command: "bash"
```

Comment:
This still ends up needing to be an object structure, so you'll want to set

```
command: if GrammarUtils.OperatingSystem.isWindows() then "cmd" else "bash"
```

followed by conditionally setting within the `args` function.